app/controllers/api/v4/facility_teleconsultations_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::V4::FacilityTeleconsultationsController < APIController
+  before_action :set_facility
+  before_action :validate_teleconsultation_facility_belongs_to_users_facility_group
+
+  attr_reader :facility
+
+  def show
+    render json: { teleconsultation_phone_number: teleconsultation_phone_number }
+  end
+
+  private
+
+  def teleconsultation_phone_number
+    "#{facility.teleconsultation_isd_code} #{facility.teleconsultation_phone_number}" if
+      facility.teleconsultation_isd_code && facility.teleconsultation_phone_number && facility.enable_teleconsultation
```

Comment:
We should provide the number even if teleconsultation is disabled, let the client decide how to manage that.